client/packages/common/src/ui/components/navigation/Tabs/DetailTabs.tsx

```diff
@@ -45,6 +50,8 @@ export const DetailTabs: FC<DetailTabsProps> = ({
   );
   useEffect(() => {
     handleResize();
+    // depending on handleResize causes too many renders
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I guess you can move the definition of handleResize out of the component to fix this?

Reply:
yes! I thought I had. thanks, will do